extensions/binding/org.eclipse.smarthome.binding.fsinternetradio/src/main/java/org/eclipse/smarthome/binding/fsinternetradio/internal/radio/FrontierSiliconRadio.java

```diff
@@ -132,8 +132,7 @@ public void setVolumeAbsolute(int volume) throws IOException {
      * @throws IOException if communication with the radio failed, e.g. because the device is not reachable.
      */
     public void setVolumePercent(int volume) throws IOException {
-        final int newVolumePercent = volume < 0 ? 0 : volume > 100 ? 100 : volume;
-        final int newVolumeAbsolute = (newVolumePercent * 32) / 100;
+        final int newVolumeAbsolute = (volume * 32) / 100;
```

Comment:
It is a public function but a bundle internal one.
So I am fine if you know that every caller already ensures (has this been checked?) that the volume is in range [0;100]
I would prefer to add this to the JavaDoc, too (e.g. @param volume the volume that must be in range [0;100]).